server/instances.go

```diff
@@ -173,24 +174,24 @@ func (p *Plugin) UninstallInstance(instanceID types.ID, instanceType InstanceTyp
 			var err error
 			instance, err = p.instanceStore.LoadInstance(instanceID)
 			if err != nil {
+				if strings.Contains(err.Error(), "not found") {
+					instances.Delete(instanceID)
+					if err = p.instanceStore.StoreInstances(instances); err != nil {
+						return err
+					}
+					if err = p.disconnectUsers(instance); err != nil {
```

Comment:
`instance` is nil right here, right?

Reply:
yeah, it should be, so there is no need to disconnect the users because we need the instance for it.